static/src/js/utils/getUiSchema.js

```diff
@@ -17,6 +18,8 @@ const getUiSchema = (conceptType) => {
       return toolsUiSchema
     case 'Variable':
       return variableUiSchema
+    case 'Keyword':
```

Comment:
I think just in your KeywordForm reference the uiScheme directly rather than grabbing it through this function, as I think its intent is just for umm schemas, see other comment.